src/core/receiver/tcp_cmd_interface.cc

```diff
@@ -55,6 +55,10 @@ TcpCmdInterface::TcpCmdInterface()
     map_signal_pretty_name_["L5"] = "L5";
     map_signal_pretty_name_["B1"] = "B1I";
     map_signal_pretty_name_["B3"] = "B3I";
+
+    map_state_name_[0] = "STBY";
+    map_state_name_[1] = "ACQ";
+    map_state_name_[2] = "TRK";
```

Comment:
Please also consider `map_state_name_[3] = "DROP";`

Reply:
Fixed in commit https://github.com/gnss-sdr/gnss-sdr/pull/419/commits/b6b7cc76a351bea1bd25d7eb37e2e441bba968a5